jans-auth-server/client/src/main/java/io/jans/as/client/RegisterRequest.java

```diff
@@ -1790,4 +1938,28 @@ public String getRedirectUrisRegex() {
     public void setRedirectUrisRegex(String redirectUrisRegex) {
         this.redirectUrisRegex = redirectUrisRegex;
     }
+
+    private void addClientMetadataParam(Map<String, String> parameters, RegisterRequestParam registerRequestParam, ClientMetadataValue clientMetadataValue) {
+        if (clientMetadataValue != null && clientMetadataValue.size() > 0) {
+            for (String languageTag : clientMetadataValue.getLanguageTags()) {
+                if (StringUtils.isBlank(languageTag)) {
+                    parameters.put(registerRequestParam.toString(), clientMetadataValue.getValue());
+                } else {
+                    parameters.put(registerRequestParam.toString() + "#" + languageTag, clientMetadataValue.getValue(languageTag));
+                }
+            }
+        }
+    }
```

Comment:
ClientMetadataValue -> LocalizedString.
We can add method to `LocalizedString` and reuse it.
```
    public Map<String, String> addToMap(Map<String, String> map, String key) {
        if (!values.isEmpty) {
            return map;
        }

        for (String languageTag : getLanguageTags()) {
            if (StringUtils.isBlank(languageTag)) {
                map.put(key, getValue());
            } else {
                map.put(key + "#" + languageTag, getValue(languageTag));
            }
        }
        return map;
    }
```